src/screens/home/components/HomeView.js

```diff
@@ -10,13 +10,18 @@ import { CozyProxyWebView } from '/components/webviews/CozyProxyWebView'
 import { consumeRouteParameter } from '/libs/functions/routeHelpers'
 import { resetUIState } from '/libs/intents/setFlagshipUI'
 import { useSession } from '/hooks/useSession'
+import { routes } from '/constants/routes'
+import { navigate } from '/libs/RootNavigation'
+import { getData, StorageKeys } from '/libs/localStore/storage'
 
 const unzoomHomeView = webviewRef => {
   webviewRef?.injectJavaScript(
     'window.dispatchEvent(new Event("closeApp"));true;'
   )
 }
 
+let hasRenderedOnce = false
```

Comment:
This is used to mitigate the side effect of not being able to initiate the Home route and the Lock route simultaneously. It works for now but it may makes harder to understand/maintain the code later. Can we keep track of this and think about how to improve this later?

Reply:
Sure, it's not ideal